src/command_modules/azure-cli-hdinsight/azure/cli/command_modules/hdinsight/_params.py

```diff
@@ -0,0 +1,70 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
```

Comment:
Remove and please break the lines at 120. We discourage this pylint disable statement.